templates/web/base/alert/index.html

```diff
@@ -30,8 +30,10 @@ <h1>[% loc('Local RSS feeds and email alerts') %]</h1>
 </p>
 <form method="get" action="/alert/list" class="form-box js-geolocate">
   <fieldset>
+    <label for="pc">[% loc('Postcode or street name and area') %]</label>
+    <p class="form-hint" id="pc-hint">[% tprintf(loc('e.g. ‘%s’ or ‘%s’'), c.cobrand.example_places) %]</p>
```

Comment:
Could form-hint be a bit grey?

Reply:
Stop that form being grey :)

Reply:
`.form-hint`s are now `#666` grey!

![screen shot 2018-08-01 at 10 24 32](https://user-images.githubusercontent.com/739624/43513229-232f39ca-9575-11e8-95b5-9204e4cfece4.png)